stories/storybook-common/api-table/api-table-components.tsx

```diff
@@ -28,11 +28,30 @@ export const Table = ({ children }: TableProps) => {
 const StyledTable = styled.table`
     width: 100%;
     position: relative;
+    border-collapse: collapse;
+    border-spacing: 0;
+    line-height: 1.5;
+
+    code {
+        background-color: #f5f5f5;
+        color: #d0021b;
+        font-size: 0.875em;
+        font-weight: 400;
+        padding: 0.25em 0.5em;
+        white-space: pre-wrap;
+    }
 
```

Comment:
Could you help me add one for `<a>` also. It's in purple at the moment

Reply:
OKie